lib/tokens/session_token.js

```diff
@@ -21,6 +21,7 @@ module.exports = function (log, inherits, Token) {
     this.emailVerified = !!details.emailVerified
     this.verifierSetAt = details.verifierSetAt
     this.locale = details.locale || null
+    this.mustVerify = details.mustVerify || false
```

Comment:
Would it be more proper to !!-coerce this to a boolean? Or is that already guaranteed by all calling cases?